app/src/main/java/com/example/pinterbest/utilities/SessionManager.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.pinterbest.utilities
+
+import android.content.Context
+import android.content.SharedPreferences
+import com.example.pinterbest.R
+
+class SessionManager(context: Context) {
```

Comment:
The manager is kind of a god object. Consider break down this class into known components, such as Interactor, Repository

Reply:
Fixed here https://github.com/enginire4ek/Pinterbest/pull/3/commits/f47063b96f0a24a56ad6def81896bbe4dee63167